### DISCUSSION TITLE
DIA-NN project deployment

### DISCUSSION THREAD

**Discussion by ylaxz** [vdemichev/DiaNN#880] [Ideas]
I want to deploy the DIA-NN project on my own computer. Is there a deployment process or running process? I haven't found a similar deployment process on GitHub. I would like to specifically examine how the data is processed step by step in this project, and how it is processed from input to output results. Do you think it is feasible to make modifications based on this?

**Comment by vdemichev** [comment]
It's very straightforward, just use the installer and then can run DIA-NN https://github.com/vdemichev/DiaNN?tab=readme-ov-file#installation
